Examples/ObjectiveC/CalloutDelegateUsageExample.m

```diff
@@ -69,13 +69,16 @@ - (void)mapView:(MGLMapView *)mapView annotation:(id<MGLAnnotation>)annotation c
 {
     // Hide the callout view.
     [self.mapView deselectAnnotation:annotation animated:NO];
+    
+    // Create an alert containing the annotation's details
```

Comment:
We might say “Show”, rather than “Create” — the presentation is the main purpose here.